### ISSUE TITLE
Link to home page not working

### ISSUE THREAD

**Issue by amatkins91** [house-of-abbey/GarminThermoNest#2]
Hi
I hope you are well.
The link to the app home page isn't working.
Is this still a live project, as I'm interested in controlling my Nest from my Garmin watch?
Thanks
Alex

**Comment by philipabbey** [comment]
Hi Alex,

It is live in that I used it personally. But getting it made public has been fraught because:

- I have to make the Google Cloud API part public which means:
   - I have to meet Google's standards for operation which I don't think were written with a watch app in mind. I've done a compliance table at https://github.com/house-of-abbey/GarminThermoNest/blob/main/docs/development.md#1-functionality.
   - I have to create a video of the application in order to apply to Google to move my API and app out of prototyping.
   - I have to figure out how cloud services work when its others not just me using the API I paid for, and I'm clueless here.
- The SDK and runtime has a bug since Google deprecated the IE browser as insecure. That move broke some of the OAuth functionality. See https://github.com/house-of-abbey/GarminThermoNest/blob/main/docs/index.md#open-authentication-oauth-for-smart-device-management-api
- OAuth is slow, making the glance view slow to show

We have just fixed the link to the 404 page as the URL must have changed, it should now be working and pointing to https://house-of-abbey.github.io/GarminThermoNest/.

So basically it has stalled for public release but I use it privately.

Regards,

Philip
